### PR TITLE
Setup-controllers

### PR DESCRIPTION
**Pull request by Reggeditt**
In this pull request;
- For each URL I create:
    - A route.
    - An action in the correct controller.
    - A view file.
- View files include only placeholders written in plain HTML

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :users
+  resources :posts
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root 'users#index'
```
Comment (mikethreels) [inline config/routes.rb]: - [ ] please make sure to not expose any unnecessary endpoints on your application for security, your routes should look something like below:
```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi @Reggeditt, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job creating the controllers.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (Reggeditt) [reply config/routes.rb:5]: thanks for pointing it out

Comment (Reggeditt) [reply config/routes.rb]: thanks for pointin it out

Comment (Reggeditt) [reply config/routes.rb]: fixed

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+  resources :users, only: [:index, :show] do
+  end
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :posts, only: [:index, :show] do
+  end
+
+
+  root 'users#index'
```
Comment (BregornOriginal) [inline config/routes.rb]: - [ ] Kinldy as the previous reviewer show you, nest your `post` routes inside your `user` page. You can follow the example provided by him.

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @Reggeditt  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

giphy (5)

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good PR description

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @Reggeditt,

## APPROVED 🟢

## Highlights
✔️ Project met all technical requirements

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**